.env.base

```diff
@@ -50,6 +50,7 @@ REACT_APP_FEATURE_PHANTOM_WALLET=true
 REACT_APP_FEATURE_THOR_FREE_FEES=true
 REACT_APP_FEATURE_LIMIT_ORDERS=false
 REACT_APP_FEATURE_NEW_TRADE_FLOW=true
+REACT_APP_FEATURE_NEW_WALLET_FLOW=false
```

Comment:
False everywhere, will be toggled on in dev as all wallets options are brought in.